cmd/webhook/cmd.go

```diff
@@ -75,6 +79,23 @@ func run() func(*cobra.Command, []string) error {
 			return err
 		}
 
+		isOpenShift := false
+
+		client, err := discovery.NewDiscoveryClientForConfig(kubeConfig)
+		if err != nil {
+			logd.Get().WithName("platform").Info("detected platform", "platform", "unknown")
```

Comment:
the message should be `failed to detect platform, due to discovery client issues`, and you could even log the error 
so `log.Error(...)`

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/4952/commits/ad4cbaec98a5337510e2f83a8aa0876c2e946262